build-logic-commons/gradle-plugin/src/main/kotlin/gradlebuild.cache-miss-monitor.gradle.kts

```
@@ -97,7 +97,8 @@ fun Project.isExpectedCompileCacheMiss() =
         "Check_CompileAllBuild",
         "Component_GradlePlugin_Performance_PerformanceLatestMaster",
         "Component_GradlePlugin_Performance_PerformanceLatestReleased",
-        "Check_Gradleception"
+        "Check_Gradleception",
+        "Gradle_Master_Check_GradleceptionWithGroovy4"
```

```suggestion
        "GradleceptionWithGroovy4"
```

To make this work with multiple branch projects in Teamcity we leave out the branch prefix here, and only check whether the given String is a suffix of the configuration name. See further down.

---

Fair, the others are also without the project specifier.